install_config/topics/configuring_for_gce_registry-proc.adoc

```diff
@@ -0,0 +1,289 @@
+
+. Take a default YAML configuration file as a base and replace the `filesystem`
+entry in the storage section with the GCP entry such as below. The resulting
+storage section may look like this:
++
+[source,yaml]
+----
+storage:
+  delete:
+    enabled: true
+  cache:
+    blobdescriptor: inmemory
+  gcs:
+    bucket: ocp-registry <1>
+    keyfile: mykeyfile <2>
+----
+<1>	Replace with the GCP bucket name.
+<2> A private service account key file in JSON format. If using the Google Application Default Credentials, do not specify the `keyfile` parameter.
++
+See the https://docs.docker.com/registry/storage-drivers/gcs/[Google Cloud Storage Driver documentation] for more information about storage driver configuration files.
+
+. Export the current *_config.yml_*:
++
+[source,bash]
+----
+$ oc get secret registry-config \
+    -o jsonpath='{.data.config\.yml}' -n default | base64 -d \
+  >> config.yml.old
+----
+
+. Edit the file to include the GCP parameters as explained before:
++
+[source,bash]
+----
+$ cp config.yml.old config.yml
+$ vim config.yml
+----
+
+. Delete the registry-config secret:
++
+[source,bash]
+----
+$ oc delete secret registry-config -n default
+----
+
+. Recreate the configmap to reference the updated configuration file:
++
+[source,bash]
+----
+$ oc create secret generic registry-config \
+    --from-file=config.yml -n default
+----
+
+. Redeploy the registry to read the updated configuration:
++
+[source,bash]
+----
+$ oc rollout latest docker-registry -n default
+----
+
+=== Verify the registry is using GCP object storage
+After a successful registry deployment, the registry `deploymentconfig` does not
+show any information if the registry is using an `emptydir` instead of GCP object storage.
+
+.Procedure
+
+To verify if the registry is using GCP object storage:
+
+. Describe the registry:
++
+[source,bash]
+----
+$ oc describe dc docker-registry -n default
+...
+Mounts:
+  ...
+  /registry from registry-storage (rw)
+Volumes:
+registry-storage:
+Type:       EmptyDir <1>
+...
+----
+<1> The temporary directory that shares a pod's lifetime.
+
+. Check is the *_/registry_* mountpoint is empty:
++
+[source,bash]
+----
+$ oc exec \
+    $(oc get pod -l deploymentconfig=docker-registry \
+    -o=jsonpath='{.items[0].metadata.name}')  -i -t -- ls -l /registry
+total 0
+----
++
+If it is empty, it is because the GCP bucket configuration is performed in the
+`registry-config` secret:
++
+[source,bash]
+----
+$ oc describe secret registry-config
+Name:         registry-config
+Namespace:    default
+Labels:       <none>
+Annotations:  <none>
+
+Type:  Opaque
+
+Data
+====
+config.yml:  398 bytes
+----
+
+. The installer creates a config.yml file with the desired configuration using the
+extended registry capabilities as seen in xref:../install_config/registry/extended_registry_configuration.html#docker-registry-configuration-reference-storage[Storage in the installation documentation]. To view the configuration file:
++
```

Comment:
including the 'storage' section where the gcs bucket configuration is stored, perform the following: